pyroll/core/unit/unit.py

```diff
@@ -33,6 +33,18 @@ class Unit(HookHost):
     surface_area = Hook[float]()
     """Surface area of workpiece within the unit."""
 
+    volume_throughput = Hook[float]()
```

Comment:
Isn't really a volume, so either include standard length of 1 or we should think about renaming